### ISSUE TITLE
"Unsatisfied dependencies. Missing component: Linux headers for running kernel, 4.19.0-14-amd64. This is a fatal error, cannot install DisplayLink Linux Software."

### ISSUE THREAD

**Issue by caelulum2** [DisplayLink/evdi#267]
Using the newest version of Debian.  debian 10.8 (x86-64), kernel: 4.19.0-14-amd64, newest drivers on official displaylink website for ubuntu

'/home/user/Downloads/DisplayLink USB Graphics Software for Ubuntu 5.1/displaylink-driver-5.1.26.run'
Verifying archive integrity... 100% All good.
Uncompressing DisplayLink Linux Driver 5.1.26 100%
DisplayLink Linux Software 5.1.26 install script called: install
Distribution discovered: Debian GNU/Linux 10 (buster)
Unsatisfied dependencies. Missing component: Linux headers for running kernel, 4.19.0-14-amd64.
This is a fatal error, cannot install DisplayLink Linux Software.

root@debian:~# apt install linux-headers-amd64
Reading package lists... Done
Building dependency tree
Reading state information... Done
linux-headers-amd64 is already the newest version (4.19+105+deb10u9).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.

root@debian:~# apt list linux-headers-4.19.0-14-amd64
Listing... Done
linux-headers-4.19.0-14-amd64/stable,stable,now 4.19.171-2 amd64 [installed,automatic]

-
I tried the suggestion from here: 

https://github.com/AdnanHodzic/displaylink-debian/issues/28

> cd /lib/modules/your_kernel_version/build
> ln -s Makefile Kconfig
> dont forget to install linux-source for your kernel
> then restart the installer

I'm now getting the following error:

```
root@debian:~# '/home/ghost/Downloads/DisplayLink USB Graphics Software for Ubuntu 5.1/displaylink-driver-5.1.26.run' 
Verifying archive integrity...  100%   All good.
Uncompressing DisplayLink Linux Driver 5.1.26  100%  
DisplayLink Linux Software 5.1.26 install script called: install
Distribution discovered: Debian GNU/Linux 10 (buster)
Installing
Configuring EVDI DKMS module
Registering EVDI kernel module with DKMS
Building EVDI kernel module with DKMS
ERROR (code 3): Failed to build evdi/5.1.26. Consult /var/lib/dkms/evdi/5.1.26/build/make.log for details..


```

**Comment by displaylink-emajewsk** [comment]
> Using the newest version of Debian. debian 10.8 (x86-64), kernel: 4.19.0-14-amd64, newest drivers on official displaylink website for ubuntu

The current released driver is 5.3.1 (https://www.displaylink.com/downloads/ubuntu). You appear to be running 5.1.

> `ERROR (code 3): Failed to build evdi/5.1.26. Consult /var/lib/dkms/evdi/5.1.26/build/make.log for details..`

Could you please show us the contents of `make.log`? It will tell us at which point it has failed. 😇

**Comment by displaylink-emajewsk** [comment]
Closing. Please create a new issue if you experience the problem again.
